docusaurus.config.js

```diff
@@ -198,7 +198,7 @@ module.exports = {
         theme: {
           customCss: require.resolve('./src/css/custom.css')
         },
-        sitempa: {
+        sitemap: {
```

Comment:
this config never works?

Reply:
It can't ;) but it is only supposed to improve performance, locally I can't see a difference. The search was fast before as well.